src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -630,6 +635,11 @@ export default class DefaultAudioVideoController
     this.signalingTask = undefined;
     this.meetingSessionContext.videoDuplexMode = SdkStreamServiceType.RX;
     if (!this.meetingSessionContext.enableSimulcast) {
+      if (this.useUpdateTransceiverController()) {
+        this.meetingSessionContext.videoUplinkBandwidthPolicy.updateTransceiverController(
+          this.meetingSessionContext.transceiverController
+        );
+      }
       this.enforceBandwidthLimitationForSender(
         this.meetingSessionContext.videoCaptureAndEncodeParameter.encodeBitrates()[0]
```

Comment:
Early return before this?

Reply:
Oops. Do a else instead since we have more statements to execute after this.